include/exadg/solvers_and_preconditioners/multigrid/multigrid_preconditioner_base.cpp

```diff
@@ -575,11 +602,14 @@ MultigridPreconditionerBase<dim, Number>::initialize_affine_constraints(
 
   DoFTools::make_hanging_node_constraints(dof_handler, affine_constraints);
   for(auto & it : dirichlet_bc)
-    VectorTools::interpolate_boundary_values(*mapping,
+  {
+    MappingQGeneric<dim> mapping_dummy(MappingQGeneric<dim>(1));
```

Comment:
```suggestion
    MappingQGeneric<dim> mapping_dummy(1);
```
(the same also in the function below.

Reply:
I changed this as suggested